resources/verify_requirements.sh

```diff
@@ -12,8 +12,8 @@ verifyRequirements() {
         exit 1
     fi
 
-    echo "checking access to the cluster and kubectl command installation"
-    kubectl get nodes > /dev/null
```

Comment:
Nice to have to check on the permissions that we need to create the ClusterRole.